### PR TITLE
Goals

### PR DESCRIPTION
**Pull request by hollesse**

### CODE REVIEW

File: help/help.go [hunk 1]
```diff
@@ -31,6 +31,9 @@ Basic Commands with Options:
     [--squash-wip]                       Squash wip commits from wip branch, maintaining manual commits
   reset
     [--branch|-b <branch-postfix>]       Set wip branch to 'mob/<base-branch>` + configuration.WipBranchQualifierSeparator + `<branch-postfix>'
+  goal 									 Gives you the current goal of your timer.mob.sh room
```
Comment (gregorriegler) [inline help/help.go:34]: whitespace is tabs and spaces here

File: goal/goal.go [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+		say.Error(err.Error())
+		exit(1)
+	}
+}
+
+func goal(configuration config.Configuration, parameter []string) error {
+	if configuration.TimerRoom == "" {
+		return errors.New("No room sepcified. Set MOB_TIMER_ROOM to your timer.mob.sh room in .mob file.")
+	}
+	if len(parameter) > 0 {
```
Comment (gregorriegler) [inline goal/goal.go:39]: could invert this if for less bumpy road

**Comment by simonharrer** [comment]
idea: in the readme group web related commands together?

**Comment by gregorriegler** [comment]
better? @simonharrer

**Pull request merged**